docs/conf/portland/2018/press-kit.rst

```diff
@@ -0,0 +1,95 @@
+:template: 2018/na/generic.html
+
+Press Kit
+=========
+
+We love that you're supporting Write the Docs by writing about the conference, sharing news about it on social media, or asking your manager if you can attend. We're trying to make it as easy as possible for you to do any of the above, so here are some sample tweets, emails, and images you can use.
+
+Sample Tweets
+-------------
+
+Use these unedited, or tweak them as needed!
+
+
+.. raw:: html
+
+  <ul>
+    <li>I&#39;m attending writethedocs in Portland, 6-8 May 2018 to discuss all things documentation. <br><a href="https://twitter.com/share?ref_src=twsrc%5Etfw" class="twitter-share-button" data-text="I&#39;m attending writethedocs in Portland, 6-8 May 2018 to discuss all things documentation." data-url="http://www.writethedocs.org/conf/portland/2018/" data-hashtags="writethedocs" data-show-count="false">Tweet</a><script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script></li>
+    <li>Join writers, programmers, librarians, scientists, developer advocates. and support humans at #writethedocs in Portland, 6-8 May 2018 to discuss all things documentation. <br><a href="https://twitter.com/share?ref_src=twsrc%5Etfw" class="twitter-share-button" data-text="Join writers, programmers, librarians, scientists, developer advocates. and support humans at #writethedocs in Portland, 6-8 May 2018 to discuss all things documentation." data-url="http://www.writethedocs.org/conf/portland/2018/" data-hashtags="writethedocs" data-show-count="false">Tweet</a><script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script></li>
+  </ul>
+
+Sample Emails
+--------------
+
+Convince Your Employer
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Do you need help justifying why your employer should send you to the Write the Docs? Take a look at the letter below and tweak it as required.
```

Comment:
"to the Write the Docs" <conference? event?> (needs a qualifier)

Reply:
just deleted the 'the'